plugin.video.jwtv-unofficial/addon.xml

```diff
@@ -1,23 +1,23 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon
   id="plugin.video.jwtv-unofficial"
-  version="1.8.2"
+  version="1.8.3"
   name="JW Broadcasting (unofficial)"
   provider-name="Jonathan DeMarks">
 
   <requires>
     <import addon="xbmc.python" version="2.1.0" />
-    <import addon="script.module.simplejson" version="3.3.0" />
   </requires>
 
   <extension point="xbmc.python.pluginsource" library="addon.py">
     <provides>video</provides>
   </extension>
 
   <extension point="xbmc.addon.metadata">
-    <summary lang="en">Access to JW Broadcasting</summary>
-    <description lang="en">This plugin allows easy access to the content found on tv.jw.org</description>
-    <disclaimer lang="en">This is not an offical plugin, please do not contact jw.org for support.</disclaimer>
+    <summary>Access to JW Broadcasting</summary>
+    <description>This plugin allows easy access to the content found on tv.jw.org</description>
+    <disclaimer>This is not an offical plugin, please do not contact jw.org for support.</disclaimer>
+    <license>Apache License, Version 2.0</license>
```

Comment:
Just for your information, you can use the news tag in the addon.xml to provide a simple way of sharing what hás changed in adition to the changelog